src/Tickets/Commerce/Status/Pending.php

```diff
@@ -165,6 +166,16 @@ public function can_apply_to( $order, $new_status ) {
 					$message = sprintf( __( 'There are no %s available at this time.', 'event-tickets' ), tribe_get_ticket_label_plural( 'unavailable_mixed' ) );
 				}
 
+				$tickets = tribe( Module::class )->get_tickets( $post->ID );
+
+				/**
+				 * Filters the unavailability message for a ticket collection
```

Comment:
Add a `@since` to the docblock. You can just have it as `@since TBD`.

Reply:
Done.